src/test/java/io/vertx/core/http/Http2Test.java

```diff
@@ -1092,4 +1095,35 @@ public void testStreamResetErrorMapping() throws Exception {
     }));
     await();
   }
+
+  @Test
+  public void testConcurrentWrite() throws Exception {
```

Comment:
can we merge those two tests in a single in HttpTest class ?

Reply:
yep, done, although there is a slight difference  between Http 1 and 2